tests/acquisition/covidcast/test_covidcast_row.py

```diff
@@ -14,7 +14,7 @@
 )
 
 # py3tester coverage target (equivalent to `import *`)
-__test_target__ = 'delphi.epidata.acquisition.covidcast.covidcast_row'
+__test_target__ = 'delphi.epidata.common.covidcast_row'
```

Comment:
can we move this file to `tests/common/test_covidcast_row.py` to match the `src/common/` structure?